src/types.ts

```diff
@@ -963,6 +993,33 @@ export type MessageFlagsFilters = QueryFilters<
     }
 >;
 
+export type FlagsFiltersOptions = {
+  channel_cid?: string;
+  message_id?: string;
+  team?: string;
+  user_id?: string;
+};
+
+export type FlagsFilters = QueryFilters<
+  {
+    user_id?:
```

Comment:
message_user_id is missing

Reply:
Didn't commit & push the changes 🤦‍♂️

Updated, please check.